### PR TITLE
Hazard Vest Buff

### PR DESCRIPTION
**Pull request by Knoxon**
Previously hazard vest pockets could only hold w_class 1 items, not i've buffed it to w_class 2 items, eg gloves and tools. Also lets you hook welding helmets on the exosuit slot

### CODE REVIEW

**Comment by Glloyd** [comment]
engineers op nerf pls :+1:

File: code/modules/clothing/suits/jobs.dm [hunk 1]
```diff
@@ -136,7 +136,7 @@
 	var/on = 1
 	allowed = list (/obj/item/device/analyzer, /obj/item/device/flashlight, /obj/item/device/multitool, /obj/item/device/pipe_painter, /obj/item/device/radio, /obj/item/device/t_scanner, \
 	/obj/item/weapon/crowbar, /obj/item/weapon/screwdriver, /obj/item/weapon/weldingtool, /obj/item/weapon/wirecutters, /obj/item/weapon/wrench, /obj/item/weapon/tank/emergency_oxygen, \
-	/obj/item/clothing/mask/gas, /obj/item/taperoll/engineering)
+	/obj/item/clothing/mask/gas, /obj/item/taperoll/engineering, /obj/item/clothing/head/welding,)
```
Comment (Glloyd) [inline code/modules/clothing/suits/jobs.dm]: Actually, wait. Remove that comma at the end and we'll be good to go.

**Comment by Knoxon** [comment]
Commaaaas

**Comment by Glloyd** [comment]
:+1: For real this time.

**Pull request merged**